html/css/main.css

```
@@ -397,6 +397,16 @@ body {
 	height: 23vh;
 }
 
+.iteminfo-content {
+    width: fit-content;
+    float: right;
+    background-color: rgba(0, 0, 0, 0.3);
+    border: 1px solid rgb(68 68 68 / 40%);
```

```suggestion
    border: 1px solid rgba(68, 68, 68, 0.4);
```
Using rgba instead of rgb